samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -1663,6 +1719,62 @@ mfxStatus CEncodingPipeline::ResetMFXComponents(sInputParams* pParams)
     return MFX_ERR_NONE;
 }
 
+mfxStatus CEncodingPipeline::AllocExtBuffers(sInputParams *pInParams)
+{
+    MSDK_CHECK_POINTER(pInParams, MFX_ERR_NULL_PTR);
+    mfxStatus sts = MFX_ERR_NONE;
+
+    bufSet* tmpForInit   = NULL;
+    mfxExtAVCRoundingOffset* m_AvcRoundingOffset = NULL;
```

Comment:
m_ is only for class members, + this definition can got o line 1748

Reply:
OK